code/src/components/Loader.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
I can't see this on the page

Reply:
It's there, I hope. We have seen it :) But maybe you can change the toggle in dev-tools to slow down the network?